### PR TITLE
Feature ms 69 create specifics secreenflows for postulants

### PR DESCRIPTION
**Pull request by ManuelLopezAnido**
Components Interview and JobOffers done. 
Making Redux and backend for Interview.
This PR in server is needed: https://github.com/BaSD-2021/Aquiles-MindSet-server/pull/40

### CODE REVIEW

File: src/Components/Postulant/Home/index.js [hunk 1]
```diff
@@ -1,9 +1,116 @@
...
+
+  useEffect(() => {
+    dispatch(getOnePostulant(postulantId));
+  }, []);
+
   return (
     <section className={styles.home}>
-      <h2>Home</h2>
+      {console.log(selectedPostulant)}
+      <div className={styles.informationField}>
```
Comment (ignacioaristo) [inline src/Components/Postulant/Home/index.js:39]: Remove this `console.log`

File: src/Components/Postulant/Interviews/index.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+const Interviews = () => {
+  const dispatch = useDispatch();
+  const interviews = useSelector((store) => store.interviews.list);
+  const Loading = useSelector((store) => store.interviews.isLoading);
+  const errMessage = useSelector((store) => store.interviews.error);
+  useEffect(() => {
+    dispatch(getInterviews());
+  }, []);
+  console.log(interviews);
```
Comment (ignacioaristo) [inline src/Components/Postulant/Interviews/index.js:17]: This too.

File: src/Components/Postulant/Interviews/index.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  const selectedPostulantId = '61a4398da318de40f22eba2c';
+  let filtredInterviews = interviews.filter((interview) => {
+    return interview.postulantId === selectedPostulantId;
+  });
+  console.log('filtrado: ', filtredInterviews);
+  const OnClickDelete = (id) => {
```
Comment (ignacioaristo) [inline src/Components/Postulant/Interviews/index.js:23]: This too.

File: src/Components/Postulant/JobOffers/index.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+import { useDispatch, useSelector } from 'react-redux';
+import { getPositions } from 'redux/positions/thunks';
+const JobOffers = () => {
+  const dispatch = useDispatch();
+  const jobOffers = useSelector((store) => store.positions.list);
+  useEffect(() => {
+    dispatch(getPositions());
+  }, []);
+  console.log(jobOffers);
+
```
Comment (ignacioaristo) [inline src/Components/Postulant/JobOffers/index.js:12]: This too.

**Review by ignacioaristo — CHANGES_REQUESTED** [review]

File: src/Components/Postulant/Home/index.js (hunk 1 above)
Comment (ManuelLopezAnido) [reply src/Components/Postulant/Home/index.js:38]: Done

File: src/Components/Postulant/Interviews/index.js (hunk 2 above)
Comment (ManuelLopezAnido) [reply src/Components/Postulant/Interviews/index.js]: Done too

File: src/Components/Postulant/Interviews/index.js (hunk 3 above)
Comment (ManuelLopezAnido) [reply src/Components/Postulant/Interviews/index.js:21]: All done

File: src/Components/Postulant/JobOffers/index.js (hunk 4 above)
Comment (ManuelLopezAnido) [reply src/Components/Postulant/JobOffers/index.js:11]: All right

**Review by ignacioaristo — APPROVED** [review]
Nice

**Review by leimeterjoaquin — APPROVED** [review]

**Pull request merged**